### PR TITLE
Draft: Fix sessions

### PR DESCRIPTION
**Pull request by mcjcloud**
Fixes the auth workflow with sessions and enable APIs to work.

## Changes

- Register and login CLI commands
- Opened #285 to address issues with Webauthn session storage
- Make use of session in highway API calls

### New Features

Calls to highway should all complete successfully without Sessions being a blocker.

## Checklist

- [x] CLI register
- [x] Browser-based auth
- [x] Unit tests
- [x] Documentation

## References *(optional)*

Fixes 
Connects 

<sub> Mention [stepsize] in a comment if you'd like to report some technical debt. See examples [here](https://app.stepsize.com/api/demo-pr-redirect).</sub>

### CODE REVIEW

File: internal/highway/main.go [hunk 1]
```diff
@@ -209,6 +209,13 @@ func setupAPI(ctx context.Context, s *HighwayServer) error {
 	s.grpcConn = conn
 	s.grpcClient = v1.NewHighwayClient(conn)
 
+	// Enable CORS
+	// The highway will not be operating under same origin outside of local env
+	s.router.Use(func(c *gin.Context) {
+		c.Writer.Header().Set("Access-Control-Allow-Origin", "*")
```
Comment (joshLong145) [inline internal/highway/main.go:215]: do we want to accept all origins? this will be a problem once we deploy highways

**Review by joshLong145 — COMMENTED** [review]
looks good, might want to make a follow up issue for writing tests once we have the shim in place for highway

File: internal/highway/main.go (hunk 1 above)
Comment (ntindle) [reply internal/highway/main.go:215]: Can all those settings just come from cli

**Review by joshLong145 — COMMENTED** [review]
I think we still need to modify FinishAccessName to return a 'Session' and not the credential standalone. we also need to associate the did with the credential.

**Pull request closed without merging**